feature_engine/variable_transformers.py

```diff
@@ -1,11 +1,14 @@
 # Authors: Soledad Galli <solegalli@protonmail.com>
 # License: BSD 3 clause
 
```

Comment:
this module was divided in submodules in the develop branch for the v1.0.0 release. So changes here need to be updated with current code architecture. But also, see my comments about type hint.